src/ameba/cli/cmd.cr

```diff
@@ -14,7 +14,7 @@ module Ameba::Cli
       raise "Invalid usage: Cannot explain an issue and autocorrect at the same time."
     end
 
-    config = Config.load opts.config, opts.colors?
+    config = Config.load path: opts.config, colors: opts.colors?, skip_reading_config: opts.skip_reading_config?
```

Comment:
No need to use named arguments here, code is pretty self-explanatory.

Reply:
Agreed, self-explanatory after do rename.